.githooks/pre-commit

```diff
@@ -1,3 +1,7 @@
+#!/bin/sh
```

Comment:
This is my fault for not adding the shebang line, a fellow unix-user forgot about their Windows-using friends...

Reply:
Thank you for the fix!